angular/karma.conf.js

```diff
@@ -0,0 +1,38 @@
+// Karma configuration file, see link for more information
```

Comment:
This file might not be needed as it only concerns tests

Reply:
Deleted this file